CharacterCreation/01_03_Races_Tundari

```diff
@@ -0,0 +1,171 @@
+
+
+===== Language: Tuunra ===== 
+
+While different clans have their own unique dialects, they are similar enough that 
+they can be understood well by all. Tuunra features drawn-out vowels, soft consonants,
+and sounds such as clicks, chirps, trills, and a low rumbling that is nearly inaudible 
+to humans. There is another integral part of Tuunra that many species do not pick up
+on, and that is an intricate system of body language that involves both tail and ears.
+A Tundari that has lost their tail may have great difficulty getting their point across
+to others of their species.
+Tundari can communicate basic ideas and general instructions noiselessly. 
+
+===== Names ===== 
+
+First names range very much as humans, as far as origin. Often having some significance 
+with the family, first names are fairly arbitrary in much of Tundari society. Tribe and 
+clan names are held the most important, and as such, are always listed first.
+Some Tundari have been exiled, lost their clan names, or adopted their own. Others have 
+adopted names from the other cultures they now live in. 
+
+                      ===== Tuunra Name Examples =====
+                              Rra’Igri Chi’ro
+                              Sang’Aal Meari
+                              Chi’Mer Brea
+                              Chi’Krou Amri
+                              A’rhe Gaur
+
+===== Available Traits =====
+
+Clawed:
+Already equipped with built-in weapons, the large claws on your forelimbs give you an 
+advantage in hand-to-hand combat. 
+[Increased Damage for Hand-to-Hand]
+
+Soft-footed:
+An ambush predator, you are built for sneaking up on those around you. You know how to 
+handle your weight and use the knowledge passed down by generations of hunters to keep 
+your presence a secret. 
+[Stealth Rolls may be Averaged]
+
+Prehensile Tail: 
+You have spent a long time using your tail to do more than just talk with. You can grab
+onto items, use it to aid in climbing, or even hang from it. 
+[Tail has ½ the carrying capacity of an arm when upright. Full when hanging/dangling]
+	Note: Tails cannot bear the hanging weight of a Large individual.
+
+Toothed: As a predatory species, you have the customary long teeth that frighten many. 
+To you, they are commonplace, but to many others, they are far from ordinary. 
+[Gain Advantage on Intimidation Rolls toward and against Biological beings]
+
+===== Clans =====
+
+===== Rra’mau =====
+
+Size: Medium/Large
+Favored Terrain: Plains
+
+Appearance: These clans tend to have coats that range from tawny to sorrel. They are 
+leaner than the Sang’mau, as a whole.
+Tribe: The most social of all the clans, Rra’mau is the face for intergalactic trade,
+policies, and general hospitality to outsiders. While the clans all meet together to 
+decide these policies beforehand, (usually) for the betterment of the species, Rra’mau
+merely figure-head the process to outsiders.
+
+===== Available Traits =====
```

Comment:
Nonblocking request: If these traits are only available to the the specific clan, can we reduce the rank of the type of heading to a section heading? (two equals signs on each side instead of 5). I think that will make the hierarchal relationship more clear.